app/src/frontend/vue/substores/storeFiles.js

```diff
@@ -114,22 +114,32 @@ export default function(ref) {
             elem.errored = true;
           }
           let process = null;
+          let cancelToken = null;
           if (state.currPath === 'upload') {
-            process = state.operationProcesses[elem.path];
+            process = state.operationUploadProcesses[elem.path];
+            if (process) {
+              console.log(process);
```

Comment:
Should these console.logs be removed?

Reply:
Yes looks like I was sloppy with my commits. I'll take another look through make sure nothing else I used for debugging slipped through.